### DISCUSSION TITLE
2015/03/25/beacon-light-information-wanted

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#14] [Announcements]
# 2015/03/25/beacon-light-information-wanted

http://localhost:4000/2015/03/25/beacon-light-information-wanted

**Comment by jengalas** [comment]
Posted by Dick Merrill
October 14, 2015 at 10:57 am
_Imported from WordPress_

I have a photo of a beacon tower showing those the lights were used. I’d like to talk with Mr. Huber.
